src/containers/Review/assignment/AssigneeDropdown.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react'
 import { Dropdown } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { AssignmentOptions } from '../../../utils/types'
 interface AssigneeProps {
-  assignmentOptions: {
-    isCompleted: boolean
-    selected: number
-    options: {
-      key: number
-      value: number
-      text: string
-    }[]
-  }
+  assignmentOptions: AssignmentOptions
   sectionCode: string
   onChangeMethod: (selected: number) => void
 }
 
 const AssigneeDropdown: React.FC<AssigneeProps> = ({ assignmentOptions, onChangeMethod }) => {
+  const { strings } = useLanguageProvider()
   const { isCompleted, options, selected } = assignmentOptions
 
   return (
     <Dropdown
       className="reviewer-dropdown"
-      options={options}
-      value={selected}
+      text={
+        options.find(({ value }) => value === selected)?.text || strings.ASSIGNMENT_NOT_ASSIGNED
+      }
       disabled={isCompleted}
-      onChange={(_: any, { value }: any) => onChangeMethod(value as number)}
-    />
+      labeled
+      scrolling
+    >
+      <Dropdown.Menu>
+        <Dropdown.Header icon="tags" content={strings.ASSIGNMENT_NOT_ASSIGNED} />
+        <Dropdown.Divider />
+        {options.map((assignee) => (
+          <Dropdown.Item
+            {...assignee}
+            onClick={(_: any, { value }: any) => onChangeMethod(value as number)}
+          />
+        ))}
+      </Dropdown.Menu>
+    </Dropdown>
```

Comment:
Very similar to what you see above for AssignAll, we should be able to combine this two components, but not right now